gulpfile.js

```diff
@@ -50,11 +52,21 @@ gulp.task('build-worker', function() {
   return build('worker.js');
 });
 
-gulp.task('lint', function() {
-  return gulp.src(['src/**/*.js', 'demo/**/*.js'])
+const lint = function(failOnError) {
+  let stream = gulp.src(['src/**/*.js', 'demo/**/*.js'])
```

Comment:
`const`

Once you decide to use `const` it turns out that you almost never use `let` outside loops.